packages/amplify-graphql-transformer-core/src/utils/directive-wrapper.ts

```diff
@@ -23,7 +22,7 @@ export class DirectiveWrapper {
   private location?: Location;
   constructor(node: DirectiveNode) {
     this.name = node.name;
-    this.arguments = (node.arguments ?? []).map(arg => new ArgumentWrapper(arg));
+    this.arguments = (node.arguments || []).map(arg => new ArgumentWrapper(arg));
```

Comment:
Why is this change necessary? What falsy values can `node.arguments` have?

Reply:
just a copy paste difference, reverted